packages/mwp-i18n/src/withIntl.jsx

```diff
@@ -25,11 +34,16 @@ export default (messages: Messages, doInjectIntl?: boolean) => (
 	}
 
 	const WithIntl = (props: Props) => {
-		const { __locale, ...other } = props;
+		const {
```

Comment:
Shouldn't requestedLanguage and __locale be the same?

Reply:
Wouldn't be good to understand why these would be different. :)

Reply:
`__locale` is a little hack that can be used to test a `withIntl`-enhanced component without using the `renderIntl` helper - I think Niko used it awhile back but I'm not sure if it's still in use. In the actual application, the `locale` prop is set exclusively by the top-level `<IntlProvider>` - it uses the same source data as `requestLanguage`, but doesn't need to be set anywhere else.

I'll go ahead and document that more fully

Reply:
Ahhh yeah, i've been eliminating uses of `__locale` as we moved tests off to enzyme. There's only 1 component left using it which we can kill the usage of :D.

Reply:
in that case, i won't document and we can clean up eventually